examples/integration_test.py

```diff
@@ -945,6 +1010,10 @@ def print_usage(exitcode, reason=None):
             print('=' * 30, 'Verifying Producer performance (without dr_cb)', '=' * 30)
             verify_producer_performance(with_dr_cb=False)
 
+        # The throttle test is utilizing the producer.
+        print('=' * 30, 'Verifying throttle_cb', '=' * 30)
```

Comment:
Since the throttle test requires manual setup (zk, whatnot) we can't have it enabled by default in the integration tests.

If you rebase this on the adminapi you can add throttle as a test mode but remove it from the default set, you would then need to do: `examples/integration_tests.py --throttle mybroker`

Reply:
ack